README.md

```diff
@@ -281,41 +362,106 @@ def Def(name, type, term):
 Here, `Nil` and `Ext` are list constructors. `Nil()` represents an empty list,
 and `Ext(x, xs)` represents the list `xs` extended with the `x` value.
 
+In Haskell,
+
+```haskell
+data Def = Def { _name :: Name, _type :: Term, _term :: Term }
+
+type Module = [Def]
+```
+
 #### 1.0.1. Evaluation
 
-Formality-Core is, essentially, the Lambda Calculus, which is just a fancy name
-for the subset of JavaScript and Python that has only functions. Its only
-primitive operation is the beta-reduction, which is just a fancy name for
-"function application". It says that, to evaluate a term in the shape
-`((x) => <body>)(<argm>)`, one must replace every occurrence of `x` by `<argm>`
-in `<body>` in a name-capture-avoiding manner. Formally:
+Formality-Core is, in essence, just a lambda calculus (e.g. the subset
+of languages like JavaScript or Python containing only anonymous functions). 
+
+Its only primitive operation is the beta-reduction (function application): To evaluate a term in the shape `((x) => <body>)(<argm>)`, one must
```

Comment:
shouldn't that `T` be lowcase? those changes are fine

Reply:
The English grammar rule is that clauses after a `:` start with uppercase. `;` is lowercase. Could add a line-break though